### PR TITLE
[ServiceNow] Support custom state field 

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4268)

## Description
Added the *Mirrored ServiceNow Ticket custom close state code* parameter

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py [hunk 1]
```diff
@@ -2362,14 +2374,25 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], params: D
...
             fields = {key: val for key, val in fields.items() if key != 'closed_at' and key != 'resolved_at'}
 
         demisto.debug(f'Sending update request to server {ticket_type}, {ticket_id}, {fields}')
         result = client.update(ticket_type, ticket_id, fields)
 
+        # Handle case of custom state doesn't exist, reverting to the original close state
+        if is_custom_close and demisto.get(result, 'result.state') != close_custom_state:
+            fields['state'] = TICKET_TYPE_TO_CLOSED_STATE[ticket_type]
+            demisto.debug('Given custom state doesn\'t exist')
+            demisto.debug(f'Sending second update request to server with default closed state: '
```
Comment (Shellyber) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:2392]: Let's combine this two into one log

**Review by Shellyber — COMMENTED** [review]
Nice Work!
Let's make sure that the docs team will go over this as well.

**Comment by ShirleyDenkberg** [comment]
@Shellyber @bziser Doc review completed.

**Review by bziser — APPROVED** [review]
Great work! 👏

**Pull request merged**